servicepulse/servicepulse-host-config.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Configuring ServicePulse Host
+summary: Describes how to configure the ServicePulse Host
+tags:
+- ServicePulse
+- Configuration
```

Comment:
i think we should remove the tag `Configuration`. 90% of the pages are about Configuration so the value is so diluted as to be pointless. thoughts?